easybuild/base/generaloption.py

```diff
@@ -182,6 +187,7 @@ class ExtOption(CompleterOption):
           - strlist, strtuple : convert comma-separated string in a list resp. tuple of strings
           - pathlist, pathtuple : using os.pathsep, convert pathsep-separated string in a list resp. tuple of strings
               - the path separator is OS-dependent
+          - urllist, urltuple: using '|', convert urlsep separated string in a list resp. tuple of strings
```

Comment:
Nit if you want to touch that: Although it is a copy of the above there is no definition of "urlsep". So maybe:

```suggestion
          - urllist, urltuple: convert string separated by '|', to a list resp. tuple of strings
```